src/components/ButtonGroup/ButtonGroup.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import ButtonGroup from './ButtonGroup';
+import Button from '../Button/Button';
+
+test('renders without crashing', () => {
+   const tree = renderer
+      .create(
+         <ButtonGroup>
+            <Button>Button 1</Button>
+            <Button>Button 2</Button>
+         </ButtonGroup>,
+      )
+      .toJSON();
+   expect(tree).toMatchSnapshot();
+});
+
+test('renders when disabled', () => {
+   const tree = renderer
+      .create(
+         <ButtonGroup disabled>
+            <Button>Button 1</Button>
+            <Button>Button 2</Button>
+         </ButtonGroup>,
+      )
+      .toJSON();
+   expect(tree).toMatchSnapshot();
+});
+
+test('renders with non-button children', () => {
+   const tree = renderer
+      .create(
+         <ButtonGroup>
+            <Button>Button 1</Button>
+            <Button>Button 2</Button>
+            <span>Span</span>
+         </ButtonGroup>,
+      )
+      .toJSON();
+
```

Comment:
This is super minor, but I'd probably remove this extra line. Just to be consistent with how it is above.